content/migration.md

```diff
@@ -4,6 +4,10 @@ This article provides a set of guidelines for migrating from Nest version 9 to v
 To learn more about the new features we've added in v10, check out this (_ARTICLE AVAILABLE SOON_).
 There were some very minor breaking changes that shouldn't affect most users - you can find the full list of them [here](https://github.com/nestjs/nest/releases/tag/v10.0.0).
 
+### Upgrading Packages
+
+The Nest CLI does not include an `update` command. While you can upgrade your packages manually, we recommend using [ncu (npm check updates)](https://npmjs.com/package/npm-check-updates).
```

Comment:
```suggestion
While you can upgrade your packages manually, we recommend using [ncu (npm check updates)](https://npmjs.com/package/npm-check-updates).
```

I think we can remove that info because the `update` command was remove in v9 so users should be used with that already